Geode.Angular/src/app/layout/navbar/navbar.component.html

```diff
@@ -4,19 +4,25 @@
             <a routerLink="/" routerLinkActive="active" [routerLinkActiveOptions]="{exact: true}">Geode</a>
         </div>
         <div class="d-flex">
-            <div *ngIf="!authService.isUserAuthorizedSignal()">
+            <div *ngIf="!authService.isUserAuthorized$()">
```

Comment:
```suggestion
            <div *ngIf="authService.isUserAuthorized$(); else unauthorizedUser">
```

`<ng-template #unauthorizedUser>
    .... // your navbar for this state
 <ng-template>`